cmd/crictl/image.go

```diff
@@ -56,6 +56,11 @@ var pullImageCommand = cli.Command{
 			Value: "",
 			Usage: "Use `USERNAME[:PASSWORD]` for accessing the registry",
 		},
+		cli.StringFlag{
```

Comment:
Should the command fail when both "creds" and "auth" are specified? If not, the usage should explain which takes precedence over the other.

Reply:
Done. Let it fail.